src/main/java/webserver/RequestHandler.java

```diff
@@ -26,24 +30,25 @@ public void run() {
             BufferedReader br = new BufferedReader(new InputStreamReader(in, "UTF-8"));
 
             String line = br.readLine();
+            logger.debug("request first line = {}", line);
 
-            String[] requestLine = line.split(" ");
-            String url = requestLine[1];
-            if (url.equals("/")) {
-                url = requestLine[1] = "/index.html";
-            }
+            if (line != null) {
+                String[] parsedUrl = RequestParser.separateUrls(line);
+                String url = userController.mapToFunctions(parsedUrl);
```

Comment:
```suggestion
                String url = userController.mapToFunctions(line);
```
이렇게 처리하고 mapToFunctions 내부에서 RequestParser 를 활용하는 방법은 어때요?